plugins/scaffolder/src/components/FileBrowser/FileBrowser.tsx

```diff
@@ -21,12 +21,13 @@ import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ChevronRightIcon from '@material-ui/icons/ChevronRight';
 import TreeItem from '@material-ui/lab/TreeItem';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   root: {
     whiteSpace: 'nowrap',
     overflowY: 'auto',
+    padding: theme.spacing(1.5),
```

Comment:
is it frowned upon to use non-integer amounts of the spacing unit? :)

Reply:
Oh, I was just inspired by a MUI docs example, something similar to [this](https://v4.mui.com/components/toggle-button/#customized-toggle-button) example, but happy to set it to a different value. Do you have a recommendation?

Reply:
I just thought these were meant to normally be kept as integers. So that the interface has consistent spacing everywhere

Reply:
Update to use 1.